package.json

```diff
@@ -59,8 +59,8 @@
     "image/**/*",
     "navigation/**/*"
   ],
-  "packageManager": "yarn@1.22.19",
+  "packageManager": "yarn@4.3.1",
```

Comment:
shouldn’t this be bun?

Reply:
"packageManager": "bun@1.1.18" is breaking yarn install, changing to bun to yarn@3.4.1 or a version that exists/is downloadable works. Previously the yarn version was included in the monorepo, if you have yarn@1.1.18 to include in the project as before, that may work. I don't know anything, I'm just guessing. Either way, this broke things.